Modules/Numerics/Statistics/test/itkImageToHistogramFilterTest2.cxx

```diff
@@ -27,11 +27,18 @@ itkImageToHistogramFilterTest2(int argc, char * argv[])
   if (argc < 3)
   {
     std::cerr << "Missing command line arguments" << std::endl;
-    std::cerr << "Usage :  " << argv[0] << " inputRGBImageFileName outputHistogramFile.txt" << std::endl;
+    std::cerr << "Usage :  " << argv[0] << " inputRGBImageFileName outputHistogramFile.txt (autoMinumumMaximum)"
```

Comment:
Optional parameters are usually in `[brackets]`, not `(parentheses)`.

Reply:
done